src/app.rs

```diff
@@ -37,25 +30,14 @@ pub mod sub_manager;
 pub mod subs;
 pub mod types;
 
-pub use builder::{
-    AfterMount, BeforeMount, Builder as AppBuilder, MountPoint, MountType, UndefinedAfterMount,
-    UrlHandling,
-};
-pub use cfg::{AppCfg, AppInitCfg};
```

Comment:
Expansion of namespaces done out of personal preference for dev-process. Can contract again if needed

Reply:
@MartinKavik You mentioned it a few times. I'll contact again as asked.